sdk/src/opentelemetry/trace/tracer.cc

```diff
@@ -0,0 +1,24 @@
+#include "opentelemetry/sdk/trace/tracer.h"
+
+namespace opentelemetry
+{
+namespace sdk
+{
+namespace trace
+{
+Tracer::Tracer(nostd::string_view name,
+               nostd::string_view version,
```

Comment:
Why does the tracer take the name and version in its constructor?

Reply:
My understanding of the named tracer functionality was to annotate all spans produced by the tracer with the name and version pair so they could be targeted individually (e.g. for shutting certain instrumentation off).

I see in the js implementation they aren't doing this, so maybe I have some misunderstanding of the feature.  I'll pull them out for now.